app/src/main/java/com/x8bit/bitwarden/ui/platform/components/content/BitwardenContentBlock.kt

```diff
@@ -50,13 +51,13 @@ fun BitwardenContentBlock(
  * Implemented to match design component.
  */
 @Composable
-fun BitwardenContentBlock(
+private fun BitwardenContentBlock(
     headerText: AnnotatedString,
     modifier: Modifier = Modifier,
     headerTextStyle: TextStyle = BitwardenTheme.typography.titleSmall,
     subtitleText: String? = null,
     subtitleTextStyle: TextStyle = BitwardenTheme.typography.bodyMedium,
-    iconPainter: Painter? = null,
+    @DrawableRes iconResource: Int? = null,
```

Comment:
Can we make sure that we add the word `vector` here since it will crash if you use another type of drawable